components/41-organisms/form/README.md

```diff
@@ -0,0 +1,95 @@
+
+#### Field messages
+
+A field message to provide help or feedback on fields or fieldsets. See the examples.
+
+There are three types of field messages:
+
+1. Normal field message
+2. Error field message
+3. Success field message
+
+A **normal field message** is used to **help** users fill in a field or fieldset. 
+
+* The text in the field message provides extra information to the user on how the field or fieldset should be filled in. It can also be used to tell to the user *why* certain information is asked.
+* Try not to specify what is already implied by the input label. Only this kind of messages to provide extra information to the user.
+* As it is intended to provide help, a normal field message is always visible from the first page view of the form.
+
+An **error field message** is used to provide **feedback** on a field or fieldset. It informs the user that there was an error in filling in the field or fieldset.
+
+* The text in the error field message tells to the user what is wrong and how to fix it.
+* As it provides feedback on user input, an error field message is **not** visible on the first page view of the form. It is only visible during or after filling in the field or fieldset or when triggered by form validation.
+
+A **success field message** is used to inform the user that the field or fieldset is **correctly filled in**.
+
+* The text in the error field message tells the user about the success.
+* As it provides feedback on user input, a success field message is **not** visible on the first page view of the form. It is only visible during or after filling in the field or fieldset or when triggered by form validation.
+
+### Form actions
+
+Every form requires at least one form action. Form actions should be placed **inside** the form component.
+
+The **primary form action** is the first action of the form and is typically the submit button of the form. A submit button is required for every form.
+
+Make the label of the submit button clear so that the user is reminder what user input is collected, for what purpose and which action is taken when clicking the submit button. Examples are: "Send message", "Subscribe", "Next", "Continue", "Confirm". The button should have the *primary* button style.
+
+A form can also have a **secondary form action**.
+
+The secundary action is typically a way to temporarily pause filling in the current form. In multi-step forms, this button is used to get the user to the previous step. An other use, could be a button that saves but not submits the input the user already gave so that the user can continue filling in the form at another time. Either way, make the label of the button clear so that the user understands what the button does. Examples are: "Previous", "Save for later". The button should have the *secondary* button style.
```

Comment:
```suggestion
The secondary action is typically a way to temporarily pause filling in the current form. In multi-step forms, this button is used to get the user to the previous step. An other use, could be a button that saves but not submits the input the user already gave so that the user can continue filling in the form at another time. Either way, make the label of the button clear so that the user understands what the button does. Examples are: "Previous", "Save for later". The button should have the *secondary* button style.
```